pkg/cmd/chat/user/user.go

```diff
@@ -0,0 +1,184 @@
+package user
+
+import (
+	"encoding/json"
+
+	stream "github.com/GetStream/stream-chat-go/v5"
+	"github.com/GetStream/stream-cli/pkg/config"
+	"github.com/GetStream/stream-cli/pkg/utils"
+	"github.com/MakeNowJust/heredoc"
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/spf13/cobra"
+)
+
+func NewCmds() []*cobra.Command {
+	return []*cobra.Command{
+		createTokenCmd(),
+		upsertCmd(),
+		deleteCmd(),
+		queryCmd()}
+}
+
+func createTokenCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create-token --user [user-id] --expiration [epoch]",
+		Short: "Create a token",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			_, secret, err := config.GetConfig(cmd).GetCredentials(cmd)
+			if err != nil {
+				return err
+			}
+
+			userID, _ := cmd.Flags().GetString("user")
+			exp, _ := cmd.Flags().GetInt("expiration")
+
+			claims := jwt.MapClaims{
+				"user_id": userID,
+			}
+			if exp > 0 {
+				claims["exp"] = exp
+			}
+
+			token, err := jwt.NewWithClaims(jwt.SigningMethodHS256, claims).SignedString([]byte(secret))
+			if err != nil {
+				return err
+			}
+
+			cmd.Printf("Token for user [%s]:\n%s\n", userID, token)
+			return nil
```

Comment:
why don't we use the SDK ?

Reply:
oh uh, I forgot it has this feature 🤦 refactoring